nixos/modules/services/networking/dnsdist.nix

```diff
@@ -35,25 +35,16 @@ in {
     };
   };
 
-  config = mkIf config.services.dnsdist.enable {
-    systemd.services.dnsdist = {
-      description = "dnsdist load balancer";
-      wantedBy = [ "multi-user.target" ];
```

Comment:
Looks like this is still needed. Even if the unit has `WantedBy=multi-user.target` it's not getting started.
I think it's due to the fact NixOS ignores the settings in the `[install]` section.

Reply:
I poked around a couple modules which use `systemd.packages` and noticed many add `WantedBy`. Is this a bug? Maybe someone like @jtojnar or @worldofpeace might know more about that... :thinking: 

Thanks for the tip!